### PR TITLE
[wip] update faq

### PR DESCRIPTION
**Pull request by ngokevin**
goals: conciseness

### CODE REVIEW

File: src/faq/index.md [hunk 1]
```diff
@@ -193,11 +276,13 @@ Take a look at this [example](http://codepen.io/bryik/pen/pyMoGb). Here we have
 [ghpull]: https://github.com/aframevr/aframe/pulls
 [github]: http://github.com/aframevr/aframe
 [glam]: https://github.com/tparisi/glam
-[guide]: ../docs/guide
+[guide]: /docs/
```
Comment (cvan) [inline src/faq/index.md]: necessary change?

**Comment by cvan** [comment]
I would consider moving a lot of the latest [`Docs > Guide > Introduction`](https://github.com/ngokevin/aframe/blob/e73cf29/docs/guide/index.md) to here.

**Comment by ngokevin** [comment]
I'll try to slim the Intro down, but I still want to get the concepts and points across on the purpose and merits of A-Frame. I'd like the FAQ to consist of really frequently asked questions that points to the right places in the documentation.

**Comment by cvan** [comment]
how much of this do you want to push out for 0.3.0?

**Comment by ngokevin** [comment]
I moved it to the `docs/`

**Pull request closed without merging**